src/world/generator/PopulationTask.php

```
@@ -29,130 +9,91 @@
+        $popChunk = $mgr->getChunk($this->chunkX, $this->chunkZ)
+            ?? throw new AssumptionFailedError("Generated chunk missing");
+        $popChunk->setPopulated();
+
+        $this->centerChunk = FastChunkSerializer::serializeTerrain($popChunk);
+        $this->adjacentChunksData = igbinary_serialize(array_map(
+            fn(Chunk $c) => $c->isTerrainDirty() ? FastChunkSerializer::serializeTerrain($c) : null,
+            $adjChunks
+        ));
+    }
+
+    private static function setOrGenerate(SimpleChunkManager $mgr, $gen, int $x, int $z, ?Chunk $chunk): Chunk {
```

PHPStan (2.1.15)

Method pocketmine\world\generator\PopulationTask::setOrGenerate() has parameter $gen with no type specified.


(missingType.parameter)